### ISSUE TITLE
The queue should display the number of responses available for the user

### ISSUE THREAD

**Issue by rleitan** [mozilla/m-response#406]
[Precondition]:
No reviews/responses available for Respond/Moderate

[STR]:
1.Go to https://mresponse-staging.herokuapp.com/dashboard/
2.Click Respond or Moderate

[Expected result]:
Reviews/Responses should be available 

[Actual result]:
Reviews/Responses are displayed in Queue
![2019-09-26_19h23_26](https://user-images.githubusercontent.com/41474886/65706415-1f36a500-e093-11e9-8a72-ddeafa3dc091.png)

Error! No reviews available in the queue.

**Comment by M-erb** [comment]
What are the conditions that need to be met for these queue numbers?

**Comment by rleitan** [comment]
> What are the conditions that need to be met for these queue numbers?

@madasan

**Comment by madasan** [comment]
The queue should be displayed according to the users' preferences (languages etc)

**Comment by johngian** [comment]
I spend some time debugging this. The issue is that the homepage just returns a count of *all* the potential reviews but the `/reviews/` API endpoint factors in languages as well. Opening a PR for that.

**Comment by viorelaioia** [comment]
I managed to reproduce this. I had several languages added to my profile. 
I navigated to Settings and removed most of them (only Spanish remained).
When I went back to Homepage, the counter for Respond section showed 1 review, but when I clicked the section, I had several reviews to respond to. See video below.

![IMG_7981](https://user-images.githubusercontent.com/5293100/67947918-48e07000-fbed-11e9-95d4-f23303e3cc62.gif)
